lib/tap_watir/element.rb

```diff
@@ -0,0 +1,35 @@
+require 'watir'
+
+module TapWatir
+  class Element
+
+    def initialize(driver,selector) #breaks PR #7
+      @driver = driver
+      @selector = selector
+    end
+
+    # should be private
+    private
+    def locate
+      @element = @driver.find_element(@selector.keys.first,@selector.values.first)
+    end
+
+    def wd
+      @element || locate
+    end
+
+    def exist?
```

Comment:
Let's implement this as a separate PR.
Also, we'll need to write specs for this.